torat_server/client_ishell.go

```diff
@@ -236,21 +241,20 @@ func (client *activeClient) Reconnect(c *ishell.Context) {
 	c.Stop()
 }
 
-// Remove remote file
-// TODO: Is this used? Where?
+// Shred a remote file
 func (client *activeClient) Shred(c *ishell.Context) {
-	args := shared.Shred{
-		Conf: shred.Conf{
-			Times:  3,
-			Zeros:  true,
-			Remove: true,
-		},
-		Path: strings.Join(c.Args, " "),
-	}
-	var r error
-	if err := client.RPC.Call("API.Shred", &args, &r); err != nil {
-		c.Println(red("[!] Could not shred path:", args.Path, err, r))
+	var s shared.Shred
+	s.Path = strings.Join(c.Args, " ")
+	s.Times = 3
+	s.Zeros = true
+	s.Remove = true
```

Comment:
Please use a struct literal with a shorthand declaration similar to before.

Reply:
Not sure what that means in this context

Reply:
See the deleted lines from 242 to 249.

In a struct literal you declare some or all field values of a struct while declaring it, see https://tour.golang.org/moretypes/5.


A shorthand declaration can be used when the type can be infered from the assigned value.

Basically this is the same as
```
var x string = "hello"
```
the shorthand form
```
x := "hello"
```

Reply:
ooooooo I see what you mean. You just want to eliminate the var with `:=`. No prob, easy fix here